docs/content/docs/3.options/10.misc.md

```diff
@@ -47,10 +47,10 @@ If it can not detect Nuxt configuration changing, you may need to run `nuxi prep
 
 ## `debug`
 
-- type: `boolean`
+- type: `boolean | 'verbose'`
```

Comment:
if we are extending for `debug` for v8, we need to back-port for `v8` branch.

Reply:
I think Nuxt3 will still be used, so we should be back-ported to v8 as well.

Reply:
I think we can make `v9` fully compatible with both Nuxt 3 and 4, and keeping the `v8` branch up to date is starting to get difficult 😅 We may need to decide to only do maintenance/hotfixes for `v8` soon..

Reply:
> keeping the v8 branch up to date is starting to get difficult 😅 We may need to decide to only do maintenance/hotfixes for v8 soon..

Yeah, you're right, the v8 branch should be hotfix only.
This `debug` options extending should only be included in the `main` branch, i.e. v9.
So, Let's remove it from the docs for v8.

Reply:
>This debug options extending should only be included in the main branch, i.e. v9.
So, Let's remove it from the docs for v8.

Done!